app/models/order.rb

```diff
@@ -1,3 +1,5 @@
+require 'pry'
+
```

Comment:
If you try to push this to heroku, it will break; heroku hates pry (I've done this)